km62/Antonov_Mykhailo/laboratory1.sql

```diff
@@ -1,2 +1,126 @@
 -- LABORATORY WORK 1
 -- BY Antonov_Mykhailo
+
+/*---------------------------------------------------------------------------
+1.Створити схему даних з назвою – прізвище студента, у якій користувач може: 
+модифікувати таблиці та вставляти дані
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+CREATE USER antonov IDENTIFIED by antonov
+DEFAULT TABLESPACE "SYSTEM"
+TEMPORARY TABLESPACE "TEMP";
+
+GRANT "CONNECT" TO antonov ;
+GRANT "DATAPUMP_IMP_FULL_DATABASE" TO antonov ;
+GRANT "IMP_FULL_DATABASE" TO antonov ;
+
+
+
+
+
+
+
+
+
+
+
+
+/*---------------------------------------------------------------------------
+2. Створити таблиці, у яких визначити поля та типи. Головні та зовнішні ключі 
+створювати окремо від таблиць використовуючи команди ALTER TABLE. 
+Студент купляє квиток на потяг.
+4 бали
+
+---------------------------------------------------------------------------*/
+--Код відповідь:
+
+
+CREATE TABLE TABLE1 
+(
+  NAME_STUDENT VARCHAR2(20) NOT NULL,
+  TICKET_NUMBER VARCHAR2(20) 
+);
```

Comment:
You use TICKET_NUMBER as primary key later. You must add NOT NULL to TICKET_NUMBER when creating table or use other row as primary key. I consider, second variant as better. You need to add two more fields.
Before altering table you need to create one more table (TRAINS) with 4 fields, using regexp.
Then, create third table, which will have links with two previous tables (TABLE1_TRAINS)